src/metabase/middleware/security.clj

```diff
@@ -73,11 +70,8 @@
                                  ;; TODO - double check that we actually need this for Google Auth
                                  "https://accounts.google.com"]
                   :style-src    ["'self'"
-                                 "'unsafe-inline'" ; needed for Google Fonts
-                                 "fonts.googleapis.com"]
+                                 "'unsafe-inline'"]
```

Comment:
@kdoh I think we only needed `'unsafe-inline'` for Google Fonts in the first place, is it possible to remove it entirely now that we're not using Googley Fonts?

Reply:
When I tried removing that as well the app didn't render at all so my guess is something needs that but I'm not entirely sure what.

Reply:
Oh ok makes sense. for `style-src` `unsafe-inline` means any `<script>` elements embedded directly in the page. So at some point it would be a minor security improvement to go enumerate those and then you can whitelist their hashes or whatever. But for now NBD